### PR TITLE
Use add_arguments instead of option_list, to support django 1.10.

### PR DESCRIPTION
**Pull request by moemen**
This issue was reported in #152 .

[BaseCommand.option_list](https://docs.djangoproject.com/en/1.8/howto/custom-management-commands/#django.core.management.BaseCommand.option_list) is dupricated since Django 1.8 and we should use [add_arguments](https://docs.djangoproject.com/en/1.8/howto/custom-management-commands/#django.core.management.BaseCommand.add_arguments) instead.

Django 1.7 is no longer supported, so we can drop support for it safely. But if you want to keep supporting it, I can find some way to do it.

Thank you.

### CODE REVIEW

**Comment by ChristopherRabotin** [comment]
Thanks for the PR. Can you just update the `.travis.yml` file and remove lines 9 and 10 (cf. https://github.com/ChristopherRabotin/bungiesearch/blob/master/.travis.yml#L9 )? That should completely remove support for Django 1.7 and make the tests pass.

File: .travis.yml [hunk 1]
```diff
@@ -7,19 +7,21 @@ env:
...
       env: DJANGO_VERSION=">=1.8,<1.9"
     - python: "3.4"
       env: DJANGO_VERSION=">=1.9,<1.10"
     - python: "3.5"
       env: DJANGO_VERSION=">=1.8,<1.9"
     - python: "3.5"
-      env: DJANGO_VERSION=">=1.9,<1.10" COVERAGE=true
+      env: DJANGO_VERSION=">=1.9,<1.10"
+    - python: "3.5"
+      env: DJANGO_VERSION=">=1.10,<1.11" COVERAGE=true
```
Comment (ChristopherRabotin) [inline .travis.yml]: Idem

**Comment by moemen** [comment]
Thank you @ChristopherRabotin .

I'll do this ASAP.

**Comment by ChristopherRabotin** [comment]
@moemen Any update on this PR? It seems like other users of bungiesearch would like this too. If you don't have the time to finish that up, I can have a look.

**Comment by moemen** [comment]
Sorry @ChristopherRabotin for being late. I'll finish your comments and push today.

**Comment by ChristopherRabotin** [comment]
Looks good. Thanks a lot @moemen !

**Pull request merged**